core-kotlin-modules/core-kotlin-concurrency-3/src/main/java/com/baeldung/comparecollectandcollectlatest/UsingCollectLatest.kt

```diff
@@ -0,0 +1,14 @@
+package com.baeldung.comparecollectandcollectlatest
+
+import kotlinx.coroutines.flow.collectLatest
+import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.runBlocking
+fun main() {
+    runBlocking {
+        flowOf(1, 2, 3).collectLatest { value ->
+                    println(value)
+        }
+    }
+}
+
+
```

Comment:
Same with the spacing here.

```suggestion
import kotlinx.coroutines.runBlocking

fun main() {
    runBlocking {
        flowOf(1, 2, 3).collectLatest { value ->
            println(value)
        }
    }
}

```